packages/website/src/pages/index.tsx

```diff
@@ -75,16 +85,22 @@ const IndexPage = props => (
     <Section>
       <Flex flexWrap="wrap">
         <Box width={[1, 1 / 2, 2 / 3]} paddingRight={[0, 'medium', 'xxLarge']}>
-          <H2>Seriously fast testing</H2>
+          <H2>Tests that are quick to write, run, and maintain</H2>
           <Text>
-            When you're trying to test from the perspective of your user, there's a lot of information coming in.
-            All the clicks, streaming updates from the server, and the hundreds (or thousands) of background
-            processes create a highly asynchronous environment. As a result, UI testing has historically been
-            time-intensive in both setup and run time.
+            Tests are tools to help us see what we don’t know; they alert us to what is broken when we thought
+            something was unaffected. The best tests swiftly and precisely diagnose the issue so teams can iterate
+            and problem-solve on the spot.
           </Text>
           <Text>
-            BigTest brings a new stateful architecture, along with several paradigm shifts that allow tests to be
-            faster to write, execute, and maintain.
+            BigTest facilitates the creation and maintenance of tests by employing Interactors. Instead of relying
+            on class-names or HTML-attributes which continuously change, Interactors are easy to use stable
+            interfaces that developers can use (and reuse) throughout their tests.
```

Comment:
`BigTest facilitates the creation and maintenance of tests by employing Interactors; Interactors are easy to use stable interfaces that developers can use (and reuse) throughout their tests as opposed to relying on class-names or HTML-attributes which continuously change.`

Reply:
> So the repetition of Interactors (even separated by a colon) will look like a typo (and again, stylistically is just horrible to encounter).  You could get around that by saying “Interactors; these are...”
> 
> More to the point, what is it that you want to emphasize in the second sentence – What Interactors can do, or what you avoid by using them?  Which ever that is, put that last in the sentence. 
> 
> I took it to be the ease of use of Interactors – the selling point here – and your colleague seemingly confirms that with his version’s “opposed to” – in fact, we can incorporate that stronger sense by slightly changing the text by adding “having to” (see sentence).
> 
> So again this is a matter of style and meaning, and I thought you got the big picture right the order in which you wrote it.

Reply:
👌. My primary concern, aside from possible grammatical errors, was that the cadence of most of the content on the landing page is too repetitive. Like reading a 20-line poem with the same rhythm. So I often prefer `{ A } + { B }` over `{ B }, { A }`.